pages/register.tsx

```diff
@@ -145,515 +72,489 @@ export default function Register() {
     return <LoadIcon width={200} height={200} />;
   }
 
+  //disables submitting form on enter key press
+  function onKeyDown(keyEvent) {
+    if ((keyEvent.charCode || keyEvent.keyCode) === 13) {
+      keyEvent.preventDefault();
+    }
+  }
+
   return (
     <div className="flex flex-col flex-grow bg-white">
       <Head>
         <title>Hacker Registration</title>
         <meta name="description" content="Register for [HACKATHON NAME]" />
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <section id="jumbotron" className="p-2">
-        <div className="max-w-4xl py-6 mx-auto">
-          <div className="text-4xl  text-left">Hacker Registration</div>
-          <div className="text-1xl my-4 font-bold font-small text-left">
+
+      <section id="jumbotron" className="p-2 px-6">
+        <div className="max-w-4xl py-6 mx-auto flex flex-col items-center">
+          <div className="registrationTitle text-4xl font-bold text-center">
+            Hacker Registration
+          </div>
+          <div className="text-1xl my-4 font-bold font-small text-center">
             Please fill out the following fields. The application should take approximately 5
             minutes.
           </div>
         </div>
       </section>
-      <section id="registration" className="m-4">
-        <div className="max-w-4xl py-4 pt-8 mx-auto text-2xl font-bold text-left">General</div>
-        <div className="max-w-4xl py-4 mx-auto">
-          <form className="max-w-4xl mx-auto">
-            <label className="text-1xl my-4 font-bold font-small text-left">
-              *Enter your first name:
-              <br />
-              <input
-                className="border min-w-full pt-3 pb-3 text-grey-darkest px-5 bg-indigo-100 rounded-md"
-                placeholder="John"
-                type="text"
-                name="firstName"
-                autoComplete="given-name"
-                required
-                value={registrationData.user.firstName}
-                onChange={(e) => updateUserData(e)}
+
+      <section className="flex justify-center">
+        <Formik
+          initialValues={{
+            id: user?.id || '',
+            user: {
+              id: user?.id || '',
+              preferredEmail: user?.preferredEmail || '',
+              firstName: user?.firstName || '',
+              lastName: user?.lastName || '',
+              permissions: user?.permissions || ['hacker'],
+            },
+            age: 18,
```

Comment:
Default should be blank. Also, age values should be integer only.

Reply:
I don't think you're allowed to enter non integer values in the text box

Reply:
You are allowed to enter non integer inputs but the form will check against it automatically and set an error to prevent it from submitting.